docker-compose.yml

```
@@ -0,0 +1,65 @@
+version: "3.9"
+
+services:
+  app:
+    build:
+      args:
+        user: opendor-dev
+        uid: 1000
+      context: ./
+      dockerfile: .docker/php/Dockerfile
+    image: php:8.0
+    restart: unless-stopped
+    working_dir: /var/www/
+    volumes:
+      - ./:/var/www
+    depends_on:
+      - db
+    networks:
+      - opendor
+
+  db:
+    image: postgres:12.7
```

The production server runs on **PostgreSQL 13**.
```suggestion
    image: postgres:13
```